tests/UI/expected-screenshots/ContainerVersion_version_some_exist_view_user.png.orig

```diff
@@ -0,0 +1,8 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
fyi here seems to be a merge conflict file which we can remove

Reply:
deleted